src/libs/ValidationUtils.js

```diff
@@ -188,9 +188,9 @@ function isValidPaypalUsername(paypalUsername) {
  * @returns {Boolean}
  */
 function meetsMinimumAgeRequirement(date) {
-    const testDate = moment(date);
-    const minDate = moment().subtract(CONST.DATE_BIRTH.MIN_AGE_FOR_PAYMENT, 'years');
-    return testDate.isValid() && testDate.isSameOrBefore(minDate, 'day');
+    const testDate = new Date(date);
+    const minDate = subYears(new Date(), CONST.DATE_BIRTH.MIN_AGE_FOR_PAYMENT);
+    return isValid(testDate) && (isSameDay(testDate, minDate) || testDate <= minDate);
```

Comment:
Should we use something similar to `isSameOrBefore` for `testDate <= minDate`?

Reply:
there is not isSameOrBefore in date-fns